framework/test/jest/specs/unit/controller/channels/child_process_channel.spec.js

```diff
@@ -0,0 +1,622 @@
+const axon = require('axon');
+const axonRpc = require('axon-rpc');
+
+jest.mock('eventemitter2');
+jest.mock('../../../../../../src/controller/event', () =>
+	jest.fn().mockImplementation(name => ({
+		name,
+		module: 'moduleAlias',
+		key: () => 'aKey',
+		serialize: () => 'serialized',
+	}))
+);
+
+axonRpc.Client = function() {
+	return {
+		call: jest.fn(),
+	};
+};
+axonRpc.Server = function() {
+	return {
+		expose: jest.fn(),
+	};
+};
+
+const EventEmitter2 = require('eventemitter2');
+const Event = require('../../../../../../src/controller/event');
+const ChildProcessChannel = require('../../../../../../src/controller/channels/child_process_channel');
+const BaseChannel = require('../../../../../../src/controller/channels/base_channel');
+
+describe('ChildProcessChannel Channel', () => {
+	const originalModuleAlias = 'moduleAlias';
+	const params = {
+		moduleAlias: originalModuleAlias,
+		events: ['event1', 'event2'],
+		actions: {
+			action1: jest.fn(),
+			action2: jest.fn(),
+			action3: jest.fn(),
+		},
+		options: {},
+	};
+	let childProcessChannel;
```

Comment:
this let should be together with below let

Reply:
Right, thanks for pointing out